stl/inc/numeric

```diff
@@ -859,7 +859,11 @@ _NODISCARD constexpr auto _Abs_u(const _Arithmetic _Val) noexcept {
 
 // FUNCTION TEMPLATE _Stl_bitscan_forward
 template <class _Unsigned>
-constexpr unsigned long _Stl_bitscan_forward(_Unsigned _Mask) noexcept {
+_NODISCARD constexpr unsigned long _Stl_bitscan_forward(_Unsigned _Mask) noexcept {
+#ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212
+    return static_cast<unsigned long>(_Countl_zero(static_cast<_Unsigned>(~_Mask));

```

Comment:
```suggestion
    return static_cast<unsigned long>(_Countl_zero(static_cast<_Unsigned>(~_Mask)));
```

Reply:
Oh my how did I not stage that change

Reply:
You were probably still reeling from the madness of discovering that unary operators perform integral promotions. This is *not* my favorite feature of C++.

Reply:
That was one of those moments where you just shut the machine down and go for a walk